src/mococrw/private/parsingUtils.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * #%L
+ * %%
+ * Copyright (C) 2020 BMW Car IT GmbH
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+#pragma once
+
+#include <mococrw/hash.h>
+#include <mococrw/openssl_wrap.h>
+#include <mococrw/key_usage.h>
+#include <mococrw/distinguished_name.h>
+#include <mococrw/sign_params.h>
+#include <mococrw/basic_constraints.h>
+
+#include <boost/property_tree/ptree.hpp>
+#include <boost/program_options.hpp>
+
+namespace pt = boost::property_tree;
```

Comment:
It is very uncommon for headers to define namespace aliases, since – as a user – you cannot get rid of these aliases once they are defined. Please remove these and but the full signature into the methods.